pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -442,15 +442,17 @@ public BrokerResponse handleRequest(JsonNode request, @Nullable RequesterIdentit
       // Table name might have been changed (with suffix _OFFLINE/_REALTIME appended)
       LOGGER.info("requestId={},table={},timeMs={},docs={}/{},entries={}/{},"
               + "segments(queried/processed/matched/consuming/unavailable):{}/{}/{}/{}/{},consumingFreshnessTimeMs={},"
-              + "servers={}/{},groupLimitReached={},brokerReduceTimeMs={},exceptions={},serverStats={},query={}", requestId,
+              + "servers={}/{},groupLimitReached={},brokerReduceTimeMs={},offlineThreadCpuTimeNs={},"
+              + "realtimeThreadCpuTimeNs={},exceptions={},serverStats={},query={}", requestId,
```

Comment:
As mentioned in the comment above, let's add new entries in the log line towards the end to keep the existing log parser tool happy

Reply:
done

Reply:
You may want to add a comment before the line, asking to add new items at the end.

Reply:
There is already such a comment at L440